src/components/ContactList/ContactList.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import styled from 'styled-components'
+import ContactListItem from '../ContactListItem/ContactListItem'
+
+export default function ContactList({ contacts }) {
+  return (
+    <StyledList>
+      {contacts.map((contactCard, index) => (
+        <ContactListItem key={index} {...contactCard} />
```

Comment:
Maybe consider using a different key. It is not recommended by react to use the index as key because the order of items may change.
https://reactjs.org/docs/lists-and-keys.html

Reply:
I absolutely agree with Marius -> it's not recommended to use the index as key. Maybe use an individual unique id or antoher value that is absolutely unique.

Reply:
Thanks for the hint! :) In a later story I will add unique IDs to the generated user profiles. I will use the IDs instead of the index.